io/net/multiport_stress.py

```diff
@@ -54,6 +55,19 @@ def setUp(self):
         self.netmask = self.params.get("netmask", default="")
         for ipaddr, interface in zip(self.ipaddr, self.host_interfaces):
             configure_network.set_ip(ipaddr, self.netmask, interface)
+        self.peer_user = self.params.get("peer_user", default="root")
+        self.peer_password = self.params.get("peer_password", '*',
+                                             default="None")
+        self.mtu = self.params.get("mtu", default=1500)
+        self.peerinfo = PeerInfo(self.peer_ips[0], peer_user=self.peer_user,
+                                 peer_password=self.peer_password)
+        for peer_ip in self.peer_ips:
+            self.peer_interface = self.peerinfo.get_peer_interface(peer_ip)
+            if not self.peerinfo.set_mtu_peer(self.peer_interface, self.mtu):
+                self.cancel("Failed to set mtu in peer")
```

Comment:
Should be Fail as against to cancel.

Reply:
before i discuss with narshiman,he suggest to keep cancel rather then fail.because its mtu_set test should not fail so.